Snowball-Finance/axial-frontend
# Liquidity deposit approval process ui integartion

To test:
1. Test the liquidity approval steps as per the figma design flow
2. Clicks on the approve tokens will open the approving component where you need to approve all tokens
3. Then clicks on the deposit button will redirect to the depositing component
4. After successful deposit, you will see the transaction success component
